docs/guide/trace-sampling.asciidoc

```diff
@@ -0,0 +1,107 @@
+
+The goal of probability sampling is to provide you with a representative set of data that allows
+you to make statistical inferences about the entire group of data.
+In other words, in most cases, you can still find anomalous patterns in your applications, detect outages, track errors,
+and lower MTTR, even when sampling at less than `100%`.
+
+[float]
+==== What data is sampled?
+
+A sampled trace retains all data associated with it.
+
+Non-sampled traces drop <<transaction-spans,`span`>> data.
+Spans contain more granular information about what is happening within a transaction,
+like external requests or database calls.
+Spans also contain contextual information and labels.
+
+Regardless of the sampling decision, all traces retain transaction and error data.
+This means the following data will always accurately reflect *all* of your application's requests, regardless of the configured sampling rate:
+
+* Transaction duration and transactions per minute
+* Transaction breakdown metrics
+* Errors, error occurrence, and error rate
+
+// To turn off the sending of all data, including transaction and error data, set `active` to `false`.
+
+[float]
+==== Sample rates
+
+What's the best sampling rate? Unfortunately, there isn't one.
+Sampling is dependent on your data, the throughput of your application, data retainment policies, and other factors.
+A sampling rate from `.1%` to `100%` would all be considered normal.
+You may even decide to have a unique sample rate per service -- for example, if a certain service
+experiences considerably more or less traffic than another.
+
+// Regardless, cost conscious customers are likely to be fine with a lower sample rate.
+
+[float]
+==== Sampling with distributed tracing
+
+The initiating service makes the sampling decision in a distributed trace,
+and all downstream services respect that decision.
+
+In each example below, `Service A` initiates four transactions.
+In the first example, `Service A` samples at `.5` (`50%`). In the second, `Service A` samples at `1` (`100%`).
+Each subsequent service respects the initial sampling decision, regardless of their configured sample rate.
+The result is a sampling percentage that matches the initiating service:
+
+image::images/dt-sampling-example.png[How sampling impacts distributed tracing]
+
+[float]
+==== APM app implications
+
+Because the transaction sample rate is respected by downstream services,
+the APM app always knows which transactions have and haven't been sampled.
+This prevents the app from showing broken traces.
+In addition, because transaction and error data is never sampled,
+you can always expect metrics and errors to be accurately reflected in the APM app.
+
+*Service maps*
+
+Service maps rely on distributed traces to draw connections between services.
+A minimum required version of APM agents is required for Service maps to work.
+See {kibana-ref}/service-maps.html[Service maps] for more information.
```

Comment:
@ogupte Can you confirm this is accurate/make any recommendations for improvement?

Please also take a look at [L33-38](https://github.com/elastic/apm-server/pull/4475/files#diff-e537cc42e5aaa8d95f8ef9b9f1e7e468d74c43e0d7869a2973789d83bf96c1c4R33-R38).

Reply:
Looks good to me!